prac_06/guitar.py

```diff
@@ -0,0 +1,28 @@
+"""Class defining a guitar."""
+
+CURRENT_YEAR = 2021     # Improvement: Use computer's internal clock
+VINTAGE_AGE = 50
+
+
+class Guitar:
+    """Represents a Guitar object."""
+
+    def __init__(self, name="", year=0, cost=0):
+        """Initialise a Guitar instance."""
+        self.name = name
+        self.year = int(year)
```

Comment:
You could simplify this line by writing it as self.year = year.

Reply:
This would work assuming the input is encased in an int() but following the assuming all input is evil rule, I put it as an int in case the user inputs a float or string that overrides the default 0 int.